From 30d5b7ddf398799652420d636ed4ae2027447107 Mon Sep 17 00:00:00 2001
From: root <root@kvm-05-guest19.lab.eng.rdu2.dc.redhat.com>
Date: Tue, 14 Jan 2025 16:12:21 -0500
Subject: [PATCH] fix macaron version error

---
 pkg/web/macaron.go | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pkg/web/macaron.go b/pkg/web/macaron.go
index 774f4d2d499..46ccd84f5eb 100644
--- a/pkg/web/macaron.go
+++ b/pkg/web/macaron.go
@@ -1,6 +1,3 @@
-//go:build go1.3
-// +build go1.3
-
 // Copyright 2014 The Macaron Authors
 //
 // Licensed under the Apache License, Version 2.0 (the "License"): you may
-- 
2.43.5

